code/game/gamemodes/miniantags/mosquito/mosquito.dm

```diff
@@ -0,0 +1,89 @@
+/mob/living/simple_animal/mosquito
+	name = "Mosquito"
+	desc = "A blood thirsty beast, the carrier of diseases"
+	icon_state = "mosquito_alive"
+	icon_living = "mosquito_alive"
+	icon_dead = "mosquito_dead"
+	icon = 'icons/mob/mosquito.dmi'
```

Comment:
```suggestion
	icon = 'icons/goonstation/mob/mosquito.dmi'
```
Goon assets have different licenses to our regular one, they have to go in a different folder.